float32 x
float32 y
float32 z
float32 speed
string speed_unit
---
int32 error_code
int32 SUCCESS = 0
int32 ERROR_POSITION_NOT_REACHABLE = 1
int32 ERROR_INVALID_SPEED = 2
int32 ERROR_EXTERNAL_ABORT = 3
